images/prune-ocp-projects/buildah.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Create the container
+container=$(buildah from registry.access.redhat.com/ubi8/ubi)
+buildah config --label io.k8s.description="OCP Project Pruner" --label io.k8s.display-name="OCP Project Pruner" --env PATH='$PATH:/usr/local/bin' $container
+buildah add $container include/prune-ocp-projects.sh /usr/local/bin/
+buildah run $container -- curl -Lo /tmp/oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/linux/oc.tar.gz
```

Comment:
Just a suggestion ... can probably combine these 3 commands into one, like this:

```
curl -L https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/linux/oc.tar.gz | tar -C /usr/local/bin/ xzf -
```